CHANGELOG.md

```diff
@@ -86,6 +86,10 @@ Adding a new version? You'll need three changes:
   from Konnect. The initial period will be used until a valid license is retrieved.
   The default values are 1m and 12h respectively.
   [#4178](https://github.com/Kong/kubernetes-ingress-controller/pull/4178)
+- Whenever the kong configuration is corrrectly built and pushed, it is stored
+  in memory and used as a fallback in case the configuration gets broken in the
+  future.
```

Comment:
Not critical, but IMO we should mention why we do this: it's specifically to address scaled or rescheduled Pods while config is broken:

```suggestion
- Store the last known good configuration. If Kong rejects the latest
  configuration, send the last good configuration to Kong instances with no
  configuration. This allows newly-started Kong instances to serve traffic even
  if a configuration error prevents the controller from sending the latest
  configuration.
```